src/plugins/menu_qrcode.py

```diff
@@ -1,20 +1,25 @@
 # -*- coding: utf-8 -*-
 """
+src/plugins/menu_qrcode.py
+==========================
+
 A menu plugin showing QR-Code for bitmessage address in modal dialog.
 """
 
+from pybitmessage.tr import _translate
+
 import urllib
 
 import qrcode
 from PyQt4 import QtGui, QtCore
 
-from pybitmessage.tr import _translate
-
 
 # http://stackoverflow.com/questions/20452486
-class Image(qrcode.image.base.BaseImage):
+class Image(qrcode.image.base.BaseImage):           # pylint: disable=abstract-method
     """Image output class for qrcode using QPainter"""
+
     def __init__(self, border, width, box_size):
+        super(Image, self).__init__()
```

Comment:
Did you test? This may break the plugin.

Reply:
Pylint gives super-init-not-called error, that why i called super class. But after testing it breaks the plugin. Will update the code. Thank you